.gitignore

```diff
@@ -138,4 +138,12 @@ dist
 
 # Ignore back-end build output
 **/node_modules/
-**/build/
\ No newline at end of file
+**/build/
+
```

Comment:
All the files ignored below should actually be pushed to the repo so please undo this